adapter.go

```diff
@@ -148,14 +151,19 @@ func NewAdapter(driverName string, dataSourceName string,params ...interface{})
 	return a, nil
 }
 
-// NewAdapterByDB obtained through an existing Gorm instance get  a adapter, specify the table prefix and the table name
+// NewAdapterByDBUseTableName obtained through an existing Gorm instance get  a adapter, specify the table prefix and the table name
 // Example: gormadapter.NewAdapterByDBUseTableName(&db, "cms", "casbin") Automatically generate table name like this "cms_casbin"
-func NewAdapterByDBUseTableName(db *gorm.DB, prefix string,tablename string) (*Adapter, error) {
+func NewAdapterByDBUseTableName(db *gorm.DB, prefix string, tablename string) (*Adapter, error) {
```

Comment:
Rename to `NewAdapterByDBWithTableName()`, arg to `tableName`

Reply:
Also, the change of this line is just for code format and I have no plans to unify code-style at this PR.